CMSgov/mint-app
# [Mint 3067]/be analytics queries

# MINT-3067

## Description

This PR's goal is to create some basic queries to expose some  queries to provide basic analytics about  MINT data. Some data will need to be further modified as we proceed to fine tune it to the final tool we are using. This however will provide a starting point to answer the questions being asked.

You will note there is not a date rang on these queries currently, but look at data for all time. Also, many of these queries might be more useful when constrained to a single model at a time, but it is possible to run across the entire data set.



## How to test this change

1.  Run each query against the database.
2. Ensure that the queries function without error.

If desired, you can run this against a fuller data set. At the very least, I suggest that you seed the database before testing the queries

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
